src/colorpicker.cpp

```diff
@@ -17,12 +17,32 @@ static constexpr int GRAB_RADIUS = 16;
 static constexpr int GRAB_SIZE = GRAB_RADIUS * 2 + 1;
 static constexpr int MAGNIFY = 4;
 
+static QScreen *findScreenAt(const QPoint &pos)
```

Comment:
I think nowadays we can use `QGuiApplication::screenAt()` instead of this function. This is going to bump the minimum Qt version to 5.10, but this is fine.